specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/examples/GetOperationStatus.json

```
@@ -0,0 +1,23 @@
+{
+    "parameters": {
+      "endpoint": "https://{exampleAppConfigurationName}.azconfig.io",
+      "api-version": "2022-11-01-preview",
+      "snapshot": "name"
```

Rename to "Prod-2022-08-01" to be similar to the existing example for creating one?
```suggestion
      "snapshot": "Prod-2022-08-01"
```

---

sure